pallets/ethereum-events/Cargo.toml

```diff
@@ -29,6 +29,7 @@ sp-staking = { default-features = false, git = "https://github.com/paritytech/su
 pallet-session = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
 sp-application-crypto = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
 pallet-avn = { default-features = false, path = "../avn" }
+pallet-ethereum-transactions = { default-features = false, path = "../ethereum-transactions" }
```

Comment:
why are we adding this dependency?

Reply:
The PublishRootContract (from ethereum-transactions) is also killed in the ethereum-events pallet. This functionality was done in response to a prior pr comment but can be changed if necessary.